src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/items/LumberAxe.java

```diff
@@ -24,32 +23,42 @@
 	public LumberAxe(Category category, ItemStack item, String id, RecipeType recipeType, ItemStack[] recipe) {
 		super(category, item, id, recipeType, recipe);
 	}
-	
+
 	@Override
 	public BlockBreakHandler getItemHandler() {
 		return (e, item, fortune, drops) -> {
 			if (SlimefunManager.isItemSimiliar(e.getPlayer().getInventory().getItemInMainHand(), getItem(), true)) {
-				if (MaterialCollections.contains(e.getBlock().getType(), MaterialCollections.getAllLogs())) {
-					List<Location> logs = new ArrayList<>();
-					TreeCalculator.getTree(e.getBlock().getLocation(), e.getBlock().getLocation(), logs);
-
-					if (logs.contains(e.getBlock().getLocation())) logs.remove(e.getBlock().getLocation());
-					for (Location b: logs) {
-						if (SlimefunPlugin.getProtectionManager().hasPermission(e.getPlayer(), b, ProtectableAction.BREAK_BLOCK)) {
-							b.getWorld().playEffect(b, Effect.STEP_SOUND, b.getBlock().getType());
-							
-							for (ItemStack drop: b.getBlock().getDrops()) {
-								b.getWorld().dropItemNaturally(b, drop);
+				if (MaterialCollections.contains(e.getBlock().getType(), MaterialCollections.getAllLogs())
+					|| MaterialCollections.contains(e.getBlock().getType(), MaterialCollections.getAllWood())
```

Comment:
Please hold onto doing that, I am actually going to push an Update to MaterialCollections that will make this a lot easier.

Reply:
Thank god